Modules/ThirdParty/HDF5/src/itkhdf5/src/itk_hdf5_mangle.h

```diff
@@ -4047,20 +3742,1430 @@ e.g. remove mangling for H5Fget_info but leave it for H5Fget_info1 and H5Fget_in
 #define H5get_alloc_stats itk_H5get_alloc_stats
 #define H5get_free_list_sizes itk_H5get_free_list_sizes
 #define H5get_libversion itk_H5get_libversion
+#define H5is_library_terminating itk_H5is_library_terminating
 #define H5is_library_threadsafe itk_H5is_library_threadsafe
 #define H5libhdf5_settings itk_H5libhdf5_settings
 #define H5open itk_H5open
 #define H5resize_memory itk_H5resize_memory
 #define H5set_free_list_limits itk_H5set_free_list_limits
-#define HDfprintf itk_HDfprintf
 #define HDrand itk_HDrand
 #define HDsrand itk_HDsrand
-#define HDstrtoll itk_HDstrtoll
 #define Nflock itk_Nflock
 #define Pflock itk_Pflock
-#define epoch_marker_class itk_epoch_marker_class
+#define VERS_RELEASE_EXCEPTIONS itk_VERS_RELEASE_EXCEPTIONS
+#define VERS_RELEASE_EXCEPTIONS_SIZE itk_VERS_RELEASE_EXCEPTIONS_SIZE
+#define _ZN2H510AbstractDsC2Ev itk__ZN2H510AbstractDsC2Ev
```

Comment:
These look like compiler mangled symbols, I am not sure they should be listed/mangled here.